### ISSUE TITLE
Issue with flarum/akismet

### ISSUE THREAD

**Issue by iamdarkle** [the-turk/flarum-flamoji#8]
Today trying to configure Akismet, I noticed that the extension had no settings, which I found quite strange, after some research it seems that the culprit is Flamoji, being this extension enabled makes the Akismet settings disappear, there is no console error to help me understand why this happens (the only message that appears is related to another extension).

![Screenshot 2022-08-31 145341](https://user-images.githubusercontent.com/15818451/187742190-b5cf5681-055f-4f64-8ffb-5eae7e70de50.png)

After disabling Flamoji, the Akismet settings appear again.

![Screenshot 2022-08-31 145828](https://user-images.githubusercontent.com/15818451/187742214-55400619-fd7c-4c9a-9c14-fdb9fe09a513.png)

**Comment by the-turk** [comment]
ookay that's strange... IIRC the Flamoji was overriding the Flarum's default extension settings page, but I'm 100% sure I've added the necessary checks for not touching other extension's pages. 

This extension needs to be updated eventually, but I can't promise a fix right now as I'm a little snowed these days 😳

**Comment by iamdarkle** [comment]
Yeah, it's pretty weird, I would have liked to help more in describing the problem but even trying debug mode didn't get anything, as luceos said on Discord it seemed to be a typical js block rendering error that should drop errors in the console, but it doesn't seem to be the case.

I'm a bit intrigued, but probably these things end up being the dumbest bugs of all, no problem mate take your time.
